priv/repo/migrations/20161220162331_trigger_user_referential_integrity.exs

```diff
@@ -0,0 +1,13 @@
+defmodule Cog.Repo.Migrations.TriggerUserReferentialIntegrity do
+  use Ecto.Migration
+
+  def change do
+    execute """
+    ALTER TABLE triggers
+    ADD CONSTRAINT triggers_as_user_fkey
+    FOREIGN KEY(as_user) REFERENCES users(username)
+    ON DELETE RESTRICT
+    ON UPDATE CASCADE
```

Comment:
This looks good, but wouldn't it be nicer to store the user id here instead? Then we wouldn't have to worry about updates.